nengo/networks/workingmemory.py

```diff
@@ -31,22 +30,20 @@ def InputGatedMemory(n_neurons, dimensions, fdbk_scale=1.0, gate_gain=10,
         nengo.Connection(net.mem.output, net.diff.input, transform=-1)
 
         # feed difference into integrator
-        with mem_config:
-            nengo.Connection(net.diff.output, net.mem.input,
-                             transform=difference_gain)
+        nengo.Connection(net.diff.output, net.mem.input,
+                         transform=difference_gain,
+                         synapse=nengo.Lowpass(difference_synapse))
```

Comment:
Just for future reference so I don't make this mistake again, my mental model of the `ptsc` is that the when you pass it a value it turns it into a `nengo.Lowpass` anyways. Looking at the code, I can't find this. What actually happens and where does it happen?